packages/components/src/organisms/Filter/FacetBooleanItem.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Badge, Checkbox, Label } from '../../'
+import { OnFacetChange } from './Filter'
+
+export interface FacetBooleanItemProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId: string
+  /**
+   * The text displayed to identify the Boolean Facet Item.
+   */
+  label: string
+  /**
+   * Value to be emitted when Checkbox is clicked.
+   */
+  value: string
+  /**
+   * Boolean that represents the Checkbox checked state.
+   */
+  selected: boolean
+  /**
+   * Counter badge shown besides the Facet Item.
+   */
+  quantity: number
+  /**
+   * ID to identify the Checkbox and corresponding label.
+   */
+  id: string
+  /**
+   * String that identifies the current Facet key.
+   */
+  facetKey: string
+  /**
+   * This function is called when `Checkbox` from the facet changes.
+   */
+  onFacetChange: OnFacetChange
+}
+
+function FacetBooleanItem({
+  testId,
+  id,
+  selected,
+  value,
+  quantity,
+  facetKey,
+  label,
+  onFacetChange,
+}: FacetBooleanItemProps) {
+  return (
+    <li key={id} data-fs-filter-list-item>
+      <Checkbox
+        id={id}
+        checked={selected}
+        onChange={() => onFacetChange({ key: facetKey, value }, 'BOOLEAN')}
+        data-fs-filter-list-item-checkbox
+        data-testid={`${testId}-accordion-panel-checkbox`}
+        data-value={value}
+        data-quantity={quantity}
+      />
+      <Label
+        htmlFor={id}
+        className="text__title-mini-alt"
```

Comment:
We need to include this class styles to `data-fs-filter-list-item-label`, since we are not using those classes.